pages/index.js

```diff
@@ -34,6 +34,7 @@ export async function getStaticProps(context) {
         include: "uid,field_image",
         sort: "-created",
         "filter[status]": "1",
+        "filter[langcode]": context.locale,
```

Comment:
This is needed because json api in drupal by default will show all content for a language. If a piece of content is translated it will be shown in the translation, otherwise in the default language. This way we only get translated content.